ts/src/pro/woo.ts

```diff
@@ -517,11 +517,15 @@ export default class woo extends wooRest {
             'cost': cost,
             'currency': this.safeString (order, 'feeAsset'),
         };
-        const price = this.safeFloat (order, 'price');
+        let price = this.safeFloat (order, 'price');
```

Comment:
@sc0Vu we should avoid `safeFloat` usage, can you replace it with `safeNumber` ?